app/views/about/index.html.erb

```diff
@@ -0,0 +1,91 @@
+<% @page_title = t('blacklight.about.page_title', :application_name => application_name) %>
+
+<%= render 'shared/start_over_row' %>
+
+<div id="content" class="col-lg-12">
+<h1 class='page-heading'><%= t('blacklight.about.title') %></h1>
+<p>
+  The Princeton University Library’s Digital Maps and Geospatial Data website
+  allows researchers to discover and access online maps and geospatial data in
+  the Princeton Maps Collection, one of the largest cartographic collections
+  at any academic institution.
+</p>
+<h2>Types of Maps found in the Portal</h2>
+
+<p>Please note that this Portal represents only a subsection of the maps that
+  Princeton University Library has in its collection. [maybe add some text about how/where to
+  locate the maps that are not discoverable in this portal?]
+
+<h3>Sanborn Maps</h3>
+
+<p>The Sanborn Map Company is the oldest mapping company in the United States.
```

Comment:
This is pretty good. Might need a few tweaks here and there. I'd get Kim's opinion on this and, maybe, whoever is responsible for Sanborns in Special Collections these days. Emma Sarconi?

Reply:
@eliotjordan sounds good. @kelea99 any thoughts on this text about Sanborn maps? I got this info from the Special Collections website, but always good to check to make sure it's up to date.

Reply:
This sounds good to me.

Reply:
@sdellis Emma is your point person